lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -89,28 +89,13 @@ private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
 
-    $drupal_user = user_load($data->uid);
-
-    if (!isset($data->field_northstar_id_value) || $data->field_northstar_id_value === 'NONE') {
-      $northstar_response = dosomething_northstar_get_northstar_user($data->uid);
-      $northstar_response = json_decode($northstar_response);
-
-      if (!empty($northstar_response->data) && !isset($northstar_response->error)) {
-        dosomething_northstar_save_id_field($drupal_user, $northstar_response);
-      }
-    }
-
-    $last_initial = substr(dosomething_helpers_extract_field_data($drupal_user->field_last_name), 0, 1);
-    if ($last_initial === FALSE) {
-      $last_initial = NULL;
-    }
-
     $this->user = [
-      'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
-      'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
-      'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
-      'country' => dosomething_helpers_extract_field_data($drupal_user->field_address),
+      'id' => NULL,
+      'drupal_id' => $data->uid,
+      'first_name' => NULL,
+      'last_initial' => NULL,
+      'photo' => NULL,
+      'country' => NULL,
     ];
```

Comment:
Cool. Are they always `null` now? When do they get populated? Is there a reason for changing from the above if those items existed? Been a few days since I looked at this and brain is kinda fuzzy on it.